### ISSUE TITLE
Unable to scroll in a Collections documents

### ISSUE THREAD

**Issue by beau-gosse** [craftsmancoding/assetmanager#41]
Vertical scrolling is not possible in Collections documents when AssetManager is installed. The content field is only ~50% visible. This doesn't happen in regular documents.

I'm running MODX 2.3.3 and the latest build of Collections. This issue is not browser-specific.

**Comment by fireproofsocks** [comment]
Yeah, this is fixed in 1.1.1, but we haven't been able to upload it to the MODX site for some reason (they're looking into it).  You can download it here (see https://github.com/craftsmancoding/assetmanager/tree/master/builds) and that should take care of it.

**Comment by beau-gosse** [comment]
@fireproofsocks Thanks for the quick follow-up. i installed the 1.1.1 via manual upload, but nothing changed.

**Comment by fireproofsocks** [comment]
Can you verify that the previous version uninstalled?  We've seen cases where they both get installed side-by side.  You may see 2 rows in the components page, or on the file system, you may see 2 similar looking folders inside of core/components/

**Comment by beau-gosse** [comment]
@fireproofsocks I checked, but it seems the package was indeed removed including from the file system. Maybe it's my environment. I'll try it from MODX Cloud and see what it gives.
